orochi/client.py

```diff
@@ -528,6 +528,9 @@ def _song_end_handler(self, signum, frame):
             except RuntimeError as e:
                 print('*** RuntimeError: {}'.format(e))
                 self.do_stop()
+            except KeyError as e:
+                print('*** KeyError: {}'.format(e))
```

Comment:
Can you maybe instead print something like this?

    print('*** Error: Missing song information in API response. Skipping mix.')

That's probably more useful to the user.